routers/web/feed/branch.go

```diff
@@ -0,0 +1,50 @@
+// Copyright 2022 The Gitea Authors. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+package feed
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	"code.gitea.io/gitea/models/repo"
+	"code.gitea.io/gitea/modules/context"
+
+	"github.com/gorilla/feeds"
+)
+
+// ShowBranchFeed shows tags and/or releases on the repo as RSS / Atom feed
+func ShowBranchFeed(ctx *context.Context, repo *repo.Repository, formatType string) {
+	commits, err := ctx.Repo.Commit.CommitsByRange(0, 10)
+	if err != nil {
+		ctx.ServerError("ShowBranchFeed %s", err)
+		return
+	}
+
+	title := fmt.Sprintf("Latest commits for branch %s", strings.TrimSpace(ctx.Repo.BranchName))
+	link := &feeds.Link{Href: repo.HTMLURL() + "/branch/" + ctx.Repo.BranchName}
```

Comment:
I guess the variables should be escaped first before putting into URL. There should be some similar examples somewhere else.

Reply:
have updated to use the existing escaped versions